DashWallet/Sources/UI/Views/BalanceView/BalanceView.swift

```diff
@@ -31,6 +31,12 @@ final class BalanceView: UIView {
             reloadView()
         }
     }
+    
+    public var tint: UIColor? {
```

Comment:
In some cases (like crowdnode) we need the balance view entirely white, so I figured we can have a global tint setting for it.